elements/timeline-view/timeline-dialog.html

```diff
@@ -28,27 +32,36 @@
                 on-response="handleSetPresetsResponse"></iron-ajax>
 
         <paper-dialog id="dialog" with-backdrop>
-            <h2>Title goes here.</h2>
+            <h2>Please select a preset</h2>
 
-            <p>Header text goes here.</p>
-
-            <paper-dialog-scrollable>
+            <paper-dialog-scrollable id="presets_show">
```

Comment:
There's more places where I used that naming convention. Gimme a minute to run through the rest of my code.